### DISCUSSION TITLE
Is there documentation for the Import feature?

### DISCUSSION THREAD

**Discussion by wwwerk** [cnwangjie/better-onetab#414] [Q&A]
What is the input file type?

Why are Import from OneTab and JSON greyed out? 

<img width="862" alt="Screen Shot 2022-01-20 at 9 12 57 AM" src="https://user-images.githubusercontent.com/10946317/150388224-92025e9c-b619-4445-90cd-dcd8c6950a02.png">

I can help write some documentation and help pages. Also can help with improving English translation in interface.

**Comment by cnwangjie** [comment]
Input is the result of export.

The button will be available after filling in the text area below. :)
